dao/src/main/java/com/fit_goal/domain/EventDto.java

```diff
@@ -0,0 +1,33 @@
+package com.fit_goal.domain;
+
+
+import lombok.*;
```

Comment:
it's a bad practice to import with *, check the intellij config joc.jenkins.homeawaycorp.com/j_acegi_security_check

Reply:
fixed